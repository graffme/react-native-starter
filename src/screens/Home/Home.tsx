import React from 'react';
import { StyleSheet, View, Text } from 'react-native';

const Home: React.FunctionComponent<any> = () => {
  return (
    <View style={styles.container}>
      <Text>React Native Starter</Text>
    </View>
  );
};

const styles = StyleSheet.create({
  container: {
    alignItems: 'center',
    flex: 1,
    justifyContent: 'center',
  },
});

export default Home;
