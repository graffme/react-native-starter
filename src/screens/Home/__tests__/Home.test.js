import React from 'react';
import Home from '../Home';
import { shallow } from 'enzyme';

describe('Home', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
  });
});
