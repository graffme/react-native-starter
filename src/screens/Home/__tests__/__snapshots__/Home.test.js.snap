// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    React Native Starter
  </Text>
</View>
`;
